src/aliases/placeholder/parser.go

```diff
@@ -0,0 +1,62 @@
+package placeholder
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+)
+
+type Parser struct {
+	regex *regexp.Regexp
+}
+
+func New() (*Parser, error) {
+	const aliasParseFormat = `{{(?m)(?P<registry>[a-zA-Z0-9-_+]+):(?P<alias>[a-zA-Z0-9-_+]+)}}$`
+	regex, err := regexp.Compile(aliasParseFormat)
+	if err != nil {
+		return nil, errors.InvalidFormatError("bad regexp format '%v': %v", aliasParseFormat, err)
```

Comment:
this seems to be a `ConfigError` instead as the regex is part of the codebase.

Reply:
Yes, I guess I couldn't see which type corresponded best. We should document what error does what.